tests/specs/feedback/feedback.ctrl.spec.js

```diff
@@ -0,0 +1,20 @@
+/*describe('Feedback Controller', function() { 
```

Comment:
Johnny is going to make some changes in the controller, so I will wait to mock it properly